integrations-and-supported-tools/pytorch/scripts/requirements.txt

```
@@ -1,4 +1,6 @@
 neptune
+neptune-pytorch
```

Nitpick, but should it be:

```suggestion
neptune[pytorch]
```